spotinst_sdk2/clients/ocean/__init__.py

```diff
@@ -457,3 +458,320 @@ def launch_nodes_in_vng(self, ocean_launch_spec_id: str, amount: int):
+            query_params=dict(aksClusterName=aks_cluster_name, resourceGroupName=resource_group_name))
+
+        formatted_response = self.convert_json(response, 
+                                               self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
+    
+    def create_ocean_vng(self, vng: azure_ocean.VirtualNodeGroupTemplate):
+        """
+        Create a VNG inside ocean cluster
+        
+        # Arguments
+        vng (VirtualNodeGroup): VirtualNodeGroup Object
+
+        # Returns
+        (Object): Ocean API response 
+        """
+        ocean = azure_ocean.VNGRequest(vng)
+
+        excluded_missing_dict = self.exclude_missing(json.loads(ocean.toJSON()))
+
+        formatted_missing_dict = self.convert_json(
+            excluded_missing_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_missing_dict)
+
+        response = self.send_post(
+            body=body_json,
+            url=self.__base_ocean_vng_url,
+            entity_name='ocean_aks_vng')
+
+        formatted_response = self.convert_json(response, 
+                                               self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
+    
+    def update_ocean_vng(self, vng_id: str, vng: azure_ocean.VirtualNodeGroupTemplate):
+        """
+        Update an existing VNG inside an Ocean Cluster 
+        
+        # Arguments
+        vng_id (String): ID of the Ocean Virtual Node Group
+        ocean (Ocean): Ocean object
+        
+        # Returns
+        (Object): Ocean VNG API response 
+        """
+        ocean = azure_ocean.VNGRequest(vng)
+
+        excluded_missing_dict = self.exclude_missing(json.loads(ocean.toJSON()))
+
+        formatted_missing_dict = self.convert_json(
+            excluded_missing_dict, self.underscore_to_camel)
+
+        body_json = json.dumps(formatted_missing_dict)
+
+        response = self.send_put(
+            body=body_json,
+            url=self.__base_ocean_vng_url + "/" + vng_id,
+            entity_name='ocean_aks_vng')
+
+        formatted_response = self.convert_json(
+            response,
+            self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
+    
+    def get_ocean_vng(self, vng_id: str):
+        """
+        Get an existing Ocean Virtual Node Group json
+
+        # Arguments
+        vng_id (String): ID of the Ocean VNG
+
+        # Returns
+        (Object): Ocean VNG API response 
+        """
+        response = self.send_get(
+            url=self.__base_ocean_vng_url + "/" + vng_id,
+            entity_name="ocean_aks_vng"
+        )
+
+        formatted_response = self.convert_json(
+            response, self.camel_to_underscore)
+
+        return formatted_response["response"]["items"][0]
+    
+    def get_all_ocean_vng(self):
+        """
+        List the configurations for all Ocean VNGs in the specified account.
```

Comment:
1. This should support ocean_id to return VNGs in a specified cluster.
2.  The documentation also need to be corrected.